frontend/packages/erd-core/src/components/ERDRenderer/columnHandleId.ts

```diff
@@ -0,0 +1,2 @@
+export const columnHandleId = (tableName: string, columnName: string) =>
+  `${tableName}-${columnName}`
```

Comment:
📝 As it was defined in several places, it was cut out into a function to ensure that it was the same logic.